tests/execution/grpc-error.md

```diff
@@ -0,0 +1,89 @@
+# Grpc datasource
+
+```protobuf @file:news.proto
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+
+package news;
+
+message News {
+    int32 id = 1;
+    string title = 2;
+    string body = 3;
+    string postImage = 4;
+}
+
+service NewsService {
+    rpc GetAllNews (google.protobuf.Empty) returns (NewsList) {}
+    rpc GetNews (NewsId) returns (News) {}
+    rpc GetMultipleNews (MultipleNewsId) returns (NewsList) {}
+    rpc DeleteNews (NewsId) returns (google.protobuf.Empty) {}
+    rpc EditNews (News) returns (News) {}
+    rpc AddNews (News) returns (News) {}
+}
+
+message NewsId {
+    int32 id = 1;
+}
+
+message MultipleNewsId {
+    repeated NewsId ids = 1;
+}
+
+message NewsList {
+    repeated News news = 1;
+}
+```
+
+```graphql @server
+schema
+  @server(port: 8000, graphiql: true)
+  @upstream(httpCache: true, batch: {delay: 10})
+  @link(id: "news", src: "news.proto", type: Protobuf) {
+  query: Query
+}
+
+type Query {
+  news: NewsData! @grpc(method: "news.NewsService.GetAllNews", baseURL: "http://localhost:50051")
+  newsById(news: NewsInput!): News!
+    @grpc(method: "news.NewsService.GetNews", baseURL: "http://localhost:50051", body: "{{args.news}}")
+}
+input NewsInput {
+  id: Int
+  title: String
+  body: String
+  postImage: String
+}
+type NewsData {
+  news: [News]!
+}
+
+type News {
+  id: Int
+  title: String
+  body: String
+  postImage: String
+}
+```
+
+```yml @mock
+- request:
+    method: POST
+    url: http://localhost:50051/news.NewsService/GetAllNews
+    body: null
+  response:
+    status: 200
+    headers:
+      grpc-status: 3
+      grpc-message: "grpc message"
+      grpc-status-details-bin: "CAMSDWVycm9yIG1lc3NhZ2UaPgordHlwZS5nb29nbGVhcGlzLmNvbS9ncmVldGluZ3MuRXJyVmFsaWRhdGlvbhIPCg1lcnJvciBkZXRhaWxz"
```

Comment:
could you please leave a comment with the initial value of this encoded string?

Reply:
done